services/ui_backend_service/data/refiner/refinery.py

```diff
@@ -84,3 +83,27 @@ async def _postprocess(self, response: DBResponse):
 
         return DBResponse(response_code=response.response_code,
                           body=body)
+
+
+def unpack_processed_value(value) -> Tuple[bool, str, int]:
+    return (list(value) + [None] * 3)[:3]
```

Comment:
maybe a comment or something, what is the value this takes? use case is quite unclear on first read.